Assignment4/FFSM-a180285/task06.py

```diff
@@ -0,0 +1,61 @@
+from rdflib import Graph, Namespace, Literal
+from rdflib.namespace import RDF, RDFS
+from rdflib import XSD
+
+VCARD = Namespace("http://www.w3.org/2001/vcard-rdf/3.0#")
+github_storage = "https://raw.githubusercontent.com/FacultadInformatica-LinkedData/Curso2021-2022/master/Assignment4/course_materials"
+
+g = Graph()
+g.namespace_manager.bind('ns', Namespace("http://somewhere#"), override=False)
+g.namespace_manager.bind('vcard', Namespace("http://www.w3.org/2001/vcard-rdf/3.0#"), override=False)
+g.parse(github_storage+"/rdf/example5.rdf", format="xml")
+
+ns = Namespace("http://somewhere#")
+g.add((ns.Researcher, RDF.type, RDFS.Class))
+for s, p, o in g:
+  print(s,p,o)
+
+# **TASK 6.1: Create a new class named "University"**
+g.add((ns.University, RDF.type, RDFS.Class))
+
+for s, p, o in g.triples((None, RDF.type, RDFS.Class)):
+  print(s,p,o)
+
+# **TASK 6.2: Add "Researcher" as a subclass of "Person"**
+g.add((ns.Researcher, RDFS.subClassOf, ns.Person))
+for s, p, o in g.triples((None, RDFS.subClassOf, ns.Person)):
+  print(s,p,o)
+
+# **TASK 6.3: Create a new individual of Researcher named "Jane Smith"**
+g.add((ns.JaneSmith, RDF.type, ns.Researcher))
+
+for s, p, o in g.triples((ns.JaneSmith, None, None)):
+  print(s,p,o)
+
+# **TASK 6.4: Add to the individual JaneSmith the fullName, given and family names**
+g.add((ns.JaneSmith, VCARD.FN, Literal("Jane Smith", datatype=XSD.string)))
+g.add((ns.JaneSmith, VCARD.Given, Literal("Jane", datatype=XSD.string)))
+g.add((ns.JaneSmith, VCARD.Family, Literal("Smith", datatype=XSD.string)))
+for s, p, o in g.triples((ns.JaneSmith, None, None)):
+  print(s,p,o)
+
+# **TASK 6.5: Add UPM as the university where John Smith works**
+g.add((ns.UPM, RDF.type, ns.University))
+
+# Create a new property. (I'm not that creative naming properties... Sorry)
+g.add((ns.worksIn, RDF.type, ns.Property))
+g.add((ns.worksIn, RDFS.domain, ns.Person))
+g.add((ns.worksIn, RDFS.range, ns.University))
+
+# Create the relationship 
+g.add((ns.JonhSmith, ns.workIn, ns.UPM))
+
+for s, p, o in g.triples((ns.worksIn, None, None)):
+  print(s,p,o)
```

Comment:
This will return the things with `worksIn` as a subject. I think you want to print out the subject using `worksIn` as a property